azure-pipelines.yml

```diff
@@ -258,40 +258,42 @@ jobs:
           testGroup: gcstress0x3-gcstress0xc
         ${{ if eq(variables['Build.DefinitionName'], 'coreclr-outerloop-gcstress-extra') }}:
           testGroup: gcstress-extra
-        ${{ if eq(variables['Build.DefinitionName'], 'coreclr-outerloop-r2r') }}:
-          testGroup: outerloop
-          readyToRun: true
-          displayNameArgs: R2R
-        ${{ if eq(variables['Build.DefinitionName'], 'coreclr-outerloop-r2r-extra') }}:
-          testGroup: r2r-extra
-          readyToRun: true
-          displayNameArgs: R2R
-
-# ReadyToRun test jobs that are triggered by default from a PR.
 
-- ${{ if and(eq(variables['System.TeamProject'], 'public'), eq(variables['Build.Reason'], 'PullRequest'), eq(variables['Build.DefinitionName'], 'coreclr-ci')) }}:
```

Comment:
Might be good to comment about why these two "template" blocks exist, or consider merging them.